apps-script/quickstart/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A simple Node.js command-line application that makes requests to the Google Apps Script API.",
   "private": true,
   "dependencies": {
-    "googleapis": "^27.0.0"
+    "googleapis": "^39.0.0"
   },
   "devDependencies": {},
-  "engines": { "node": ">=6.4.0" },
+  "engines": {
+    "node": ">=6.4.0"
```

Comment:
Here and elsewhere, bump this up to 8.x.

The 6.x line hit EOL in April and is no longer supported. May even want to jump to 10.x since 8.x is due to be EOL'd later this year as well.

Reply:
Issue #28 is tracking the upgrade to newer versions of Node, and the adoption of async/await. I'd prefer to make that a separate effort, since it's a bit more involved.